demo/demo_twisted.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import print_function
-from ldclient.twisted import TwistedLDClient
+from ldclient.twisted_impls import TwistedLDClient
 from twisted.internet import task, defer
 
+
 @defer.inlineCallbacks
-def main(reactor):
-    apiKey = 'whatever'
-    client = TwistedLDClient(apiKey)
+def main(_):
+    api_key = 'whatever'
+    client = TwistedLDClient(api_key)
```

Comment:
Given the current design, this won't work-- this needs to be an `LDClient` with a `TwistedConfig`